updated UX for videos section, now have responsive css slider

kkemple opened the pull request:
addresses issue #96 
I updated the slider section to be a responsive css3 slider, also made it much more apparent that there are multiple videos, see screenshots below

> NOTE: the arrows for the slider nav are .png files, I know @paulfalgout has done a lot of work with the svg changes so this will need that treatment as well, but its a good start

![slider-1](https://cloud.githubusercontent.com/assets/3629876/5663923/067748c0-971a-11e4-8f2d-20e1dcc7d465.jpg)

![slider-2](https://cloud.githubusercontent.com/assets/3629876/5663922/067728c2-971a-11e4-9400-0a2cfe7a6a55.jpg)

![slider-3](https://cloud.githubusercontent.com/assets/3629876/5663924/067cb4fe-971a-11e4-99fd-0b4f168a61f4.jpg)

![slider-in-action](https://cloud.githubusercontent.com/assets/3629876/5663925/06835f66-971a-11e4-934e-d87b0f75db91.gif)

kkemple commented:
third image is when slider is below minimum breakpoint vid is 100%

kkemple commented:
also, i hid the big block of text (figcaption), i wasn't sure how that would play in to new design, i don't feel we need it with the new design, I can remove (or make title or something) and update PR if needed

paulfalgout commented:
I :heart: this.  
:wave: bye-bye to jquery-cycle.

paulfalgout commented:
So the two biggest things for the merge are likely going to be:
https://github.com/marionettejs/marionettejs.com/pull/101/files#diff-7a7786791dd24256a89805479916cfddR8
and
https://github.com/marionettejs/marionettejs.com/pull/101/files#diff-b8b2c04c1ffbd5a7760c721840d38af1R131

As for the prev/next arrows, you could go ahead and inline svg them.  Even if we end up reusing that arrow, inlined it's soooo small:

```sass
.prev {
      left: 0;
      background: $mint url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 768 768"><path color="currentColor" d="M493.28 237.28l-45.28-45.28-192 192 192 192 45.28-45.28-146.72-146.72z"></path></svg>') no-repeat 45% center;
    }

    .next {
      right: 0;
      background: $mint url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 768 768"><path color="currentColor" d="M320 192l-45.28 45.28 146.72 146.72-146.72 146.72 45.28 45.28 192-192z"></path></svg>') no-repeat 55% center;
    }
```

samccone commented on src/js/script.js:
```diff
@@ -1,11 +1,57 @@
+    // ---
+
+    (function() {
+        var $slideshowWrap = $('.video_slideshow_scroller'),
```
gosh at some point we really should use marionette ha

samccone replied:
another refactor tho

kkemple replied:
yeah we really should dog food the site, at least the homepage

kkemple replied:
the slider screams collection/composite view lol

samccone commented on src/js/script.js:
```diff
@@ -1,11 +1,57 @@
+            var distance = (-pos * ($slide.width() + parseInt($slide.css('margin-right').replace(/px/g, ''), 10)));
+            $slider.css('transform', 'translate3d(' + distance + 'px,0,0)');
+
+            if (!pos) {
```
`$slideshowPrev[pos ? 'show' : 'hide']()`

paulfalgout replied:
alternatively `$slideshowPrev.toggle(pos);`

samccone commented on src/js/script.js:
```diff
@@ -1,11 +1,57 @@
+                $slideshowPrev.show();
+            }
+
+            if (pos === slideCount - 1) {
```
ditto ^

samccone commented:
we will want to drop the script tag for cycle also right?

kkemple commented:
yeah all good, i was in a bit of rush to get to work, just wanted to get PR up before someone else did the work also. I will clean and refactor this afternoon. Let me know if you think of anything else!

kkemple commented:
@samccone @paulfalgout okay so i updated the arrows to use inline svgs, removed the pngs, refactored the js around showing and hiding the buttons, removed script tag for cycle slider, and removed commented out initialization for cycle slider

:shipit:

samccone commented:
sweeeet just testingreal quick

jasonLaster commented on src/js/script.js:
```diff
@@ -1,11 +1,40 @@
+        $(window).on('resize', _.throttle(setTransform, 300));
+        $slideshowNext.on('click', slideshowNext);
+        $slideshowPrev.on('click', slideshowPrev);
+    })();
```
Good code!

kkemple replied:
wow thanks! :blush:

samccone commented:
amazing

jmeas commented:
Nice work @kkemple!

kkemple commented:
glad to help out! :smile:
